src/renderer/services/binance/service.ts

```diff
@@ -272,16 +272,10 @@ const loadTxs$ = ({
     )
   )
 
-  const endTime = Date.now()
-  // 90 day window - similar to ASGARDEX wallet approach,
-  // see https://gitlab.com/thorchain/asgard-wallet/-/blob/develop/imports/api/wallet.js#L39-48
-  const diffTime = 90 * 24 * 60 * 60 * 1000
-  const startTime = endTime - diffTime
```

Comment:
No need to add 90 day window - it is provided [in `xchain-binance` by default](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-binance/src/client.ts#L217-L220)